soh/soh/Enhancements/randomizer/3drando/fill.cpp

```diff
@@ -1074,6 +1074,15 @@ int Fill() {
       if (ShuffleMerchants.Is(SHUFFLEMERCHANTS_HINTS)) {
         CreateMerchantsHints();
       }
+      if (GanondorfHintText) {
+        CreateGanonText();
```

Comment:
This func generates both the ganondorf light arrow hint, and the silly no-hint text. If this is off, then ganondorf will still have an empty text box.

Maybe either need to always execute this func so that the no-hint text is available and the RSK check in OTRGlobals will prevent the light arrow hint from playing. Or need to play the original ganondorf text when the RSK value is off.

Reply:
I'll let this always load, literally "for the funny".